spec/unit/shoulda/matchers/active_model/validate_presence_of_matcher_spec.rb

```diff
@@ -39,6 +39,23 @@
     end
   end
 
+  context 'a belongs_to association with a presence validation and allow_nil' do
```

Comment:
Actually, I'm noticing now that we don't seem to have tests for a `belongs_to` association at all. Can we two tests -- one for `belongs_to`, the other for `allow_nil` (doesn't have to be a `belongs_to`)?